sourced/ml/cmd_entries/repos2ids.py

```diff
@@ -1,28 +1,20 @@
 import logging
-from typing import NamedTuple
 from uuid import uuid4
 
 from sourced.ml.transformers import Ignition, Content2Ids, ContentExtractor, \
-    ContentProcess, HeadFiles, Cacher
+    ContentProcess, HeadFiles, Cacher, CsvSaver
 from sourced.ml.utils import create_engine
 
 
 def repos2ids_entry(args):
     log = logging.getLogger("repos2ids")
     engine = create_engine("repos2ids-%s" % uuid4(), **args.__dict__)
 
-    ids = Ignition(engine) \
+    Ignition(engine) \
```

Comment:
This means we can write
```
Ignition(create_engine("repos2ids-%s" % uuid4(), **args.__dict__))
```
and remove `log` because it is not needed.

Reply:
let's keep `create_engine` separately. same as the comment below:  https://github.com/src-d/ml/pull/205#discussion_r178817634